src/core/utils/cli-config.ts

```diff
@@ -47,22 +48,37 @@ export function matchLoadedConfigName(name: string) {
  * @returns An object with the `{@link SWACLIOptions}` config or an empty object if the config file, or the config entry were not found.
  */
 export async function getConfigFileOptions(configName: string | undefined, configFilePath: string): Promise<SWACLIConfig> {
-  logger.silly(`Getting config file options from ${configFilePath}...`);
+  logger.silly(`Getting config file options from "${configFilePath}"...`);
 
-  configFilePath = path.resolve(configFilePath);
-  if (!swaCliConfigFileExists(configFilePath)) {
-    logger.silly(`Config file does not exist at ${configFilePath}`);
-    return {};
+  let resolvedConfigFilePath = path.resolve(configFilePath);
+  if (!swaCliConfigFileExists(resolvedConfigFilePath)) {
+    logger.silly(`Config file does not exist at "${configFilePath}"`);
+
+    // Handle the case when the user runs the command outside the project path
+    if (!configName || !swaCliConfigFileExists(path.resolve(configName, configFilePath))) {
+      return {};
+    }
+    logger.warn(`WARNING: Config file does not exist at "${configFilePath}", but can be detected at "${path.join(configName, configFilePath)}".`);
```

Comment:
It's a little strange that both `path.join` and `path.resolve` were used in the same part of code.  I believe using only one of them is enough depending on the case.

Nit: make it as a variable in the beginning.